### PR TITLE
Added license badge in readme file!

### LINKED ISSUE #9: add license badge
Add the below code in README with other badges

```
[![LICENSE](https://img.shields.io/npm/l/fastret-framework.svg)](https://github.com/arshadkazmi42/fastret-framework/LICENSE)
```

### PR DESCRIPTION
**Pull request by atjustbeinghumaid**
Hey I've added what you wanted!
Is it fine or do I have to make any other changes?
Fixes #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # fastret-framework
 
 [![Build Status](https://travis-ci.com/arshadkazmi42/fastret-framework.svg?branch=master)](https://travis-ci.com/arshadkazmi42/fastret-framework)
-
+[![LICENSE](https://img.shields.io/npm/l/fastret-framework.svg)](https://github.com/arshadkazmi42/fastret-framework/LICENSE)
 Fastest rest api test suite builder
```
Comment (arshadkazmi42) [inline README.md]: Can you add a line break here

Comment (atjustbeinghumaid) [reply README.md]: will do it

**Comment by atjustbeinghumaid** [comment]
Is it done?

**Comment by arshadkazmi42** [comment]
@atjustbeinghumaid Congrats on landing your first PR.

**Comment by atjustbeinghumaid** [comment]
Thank you so much

**Pull request merged**